features/manage_categories.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Manage Categories
+  In order to better organize articles
+  As an admin
+  I want to add or edit a category
+
+  Background:
+    Given the blog is set up
+    And I am logged into the admin panel
+
+  Scenario: Category page is shown
+    Given I am on the admin page
+    When I follow "Categories"
+    Then I should be on the category page
+
+  Scenario: Create and edit category
```

Comment:
You read my mind and I was going to leave a comment about that.  I wasn't sure how to write an Edit scenario, or rather how to create a category instance that it could run on.  I'll do some more googling.